client/components/suggestion/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+import omit from 'lodash/omit';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import Button from 'components/button';
+import Summary from 'components/summary';
+import { translate as __ } from 'lib/mixins/i18n';
+
+const RadioButton = ( props ) => {
+	return (
+		<FormLabel>
+			<FormRadio { ...omit( props, 'children' ) } />
+			{ props.children }
+		</FormLabel>
+	);
+};
+
+const Suggestion = ( { acceptSuggestion, formValues, formActions, layout, suggestions } ) => {
+	return (
+		<div>
+			<RadioButton
+				checked={ ! acceptSuggestion }
+				onChange={ () => formActions.setFormProperty( 'acceptSuggestion', false ) } >
```

Comment:
See my answer above. This is not just a component that cares about displaying a couple of addresses. It's a component that implement the entire logic of suggestions (present the 2 options, modify the state of the flow if the user chooses any of them or goes back), so it makes sense that the component can actually call Redux actions.